pkg/azure/aks/aks.go

```diff
@@ -27,19 +59,34 @@ var (
 
 // Prometheus Metrics
 var (
-// TODO - define Prometheus Metrics
+	InstanceCPUHourlyCostDesc = prometheus.NewDesc(
+		prometheus.BuildFQName(cloudcost_exporter.MetricPrefix, subsystem, "instance_cpu_usd_per_core_hour"),
```

Comment:
Nothing for your PR, just thinking out loud:

GKE/EKS/AKS all have the same suffix(`instance_cpu_us_per_cour_hour`) and help text. Would it be worthwhile to do either of the options?
1. Define a const `instance(Cpu|Ram)MetricSuffix`that's shared across all providers? 
2. A `GenerateDesc(prefix, subsystem, suffix string) *prometheus.NewDesc` method to keep things aligned?

Reply:
@the-it do you have any strong feelings about this since you helped craft the metric documentation?

Reply:
https://github.com/grafana/cloudcost-exporter/issues/222 Created this issue to capture this thought